### ISSUE TITLE
Request for access to dental segmentation dataset

### ISSUE THREAD

**Issue by tst482** [gaudot/SlicerDentalSegmentator#39]
Hello Mr. Gauthier ,  my name is Sergii

I found your dataset/model Dataset112_DentalSegmentator_v100.zip among your works and was really impressed by the quality of the annotations:

upper skull
mandible
upper teeth
lower teeth
mandibular canal
I would also like to add sinus segmentation, so I wanted to ask whether it is possible to download the original source files somewhere in order to extend the dataset and retrain the model.

I also looked at the ToothFairy2 dataset, but the maxilla and mandible annotations there seem to be somewhat inaccurate. Your annotations are significantly better.

Thank you in advance for your response and for your excellent work.

Best regards

**Comment by GauthierDot** [comment]
Hi Sergii

Thank you for your interest. 
Infortunately we cannot share the CT / CBCT data due to data privacy issues. Which label would you like to add? Did you have a look at finetuning possibilities?

Best
Gauthier

**Comment by tst482** [comment]
Hello Gauthier,

Thank you for your response.

I am mainly interested in adding sinus segmentation to the model. I have looked at the ToothFairy2 and ToothFairy3 datasets, but unfortunately their data is based on cropped DICOM scans. Your skull and mandible annotations are much more suitable for my research and experiments.

With your permission, I would like to try fine-tuning your model by adding sinus labels using the available ToothFairy annotations.

Thank you again for your excellent work and for taking the time to reply.

Best regards,
Sergii

**Comment by GauthierDot** [comment]
Hello Sergii

Thanks for your reply.

You can try to use our shared model weights to finetune the model following nnUNet recommendations.

Another workaround could be to train a network dedicated to sinus segmentation and segment your data with a pipeline containing the 2 models inference (DentalSegmentator + your dedicated model).

Please let us know how that works as several people have expressed their interest for sinus segmentation.

Best
Gauthier

**Comment by tst482** [comment]
Thank you very much for the suggestions.
I will try and will definitely share the results once I have something working.

Best regards,
Sergii
